docs/readme.md

```diff
@@ -0,0 +1,17 @@
+# WooCommerce Blocks Handbook
```

Comment:
I feel it would be great to have this file linked from `README.md` so it gets more visibility and can be easily found from the repo front page.